about.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon"  type="image/png" href="/images/favicon.png">
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,900;1,400;1,700&display=swap" rel="stylesheet">
+  <title>Conference - Women in Analytics | About</title>
+</head>
+<body>
+  <header>
+    <nav class="desktop-navigation">
+      <ul class="settings">
+        <li>
+          <a href=""><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="">English</a>
+        </li>
+        <li>
+          <a href="">My Page</a>
+        </li>
+        <li>
+          <a href="">Logout</a>
+        </li>
+      </ul>
+
+      <ul class="desktop-menu" id="desktop-menu">
+        <li><a href="/index.html"><img src="./images/wia-logo.png" alt=""></a></li>
+        <li><a href="/index.html">Home</a></li>
+        <li><a href="/about.html" class="current">About</a></li>
+        <li><a href="/about.html#join">Join</a></li>
+        <li><a href="/index.html#partner">Partner</a></li>
+        <li><a href="#">WIA Campain</a></li>
```

Comment:
- [ ] In local the navigation is working, but in the live version it's not working . [Hint] It's happening because of `/`, since it is looking from root. I think removing `/` would work in this case.